src/components/m-dropdown/index.scss

```diff
@@ -90,6 +101,21 @@
   @include dropdown--sm();
 }
 
+
+.m-dropdown__list--native {
```

Comment:
It was not the goal to have unnecessary DOM.
Though that will work too

Reply:
@AndyOGo this is scss so i cant see what you mean with "unnecessary DOM"

Reply:
@LucaMele 
This whole code block relates to some Javascript too `DeviceStateObserver` and is also discussed above.

Reply:
ahhhhhhh ok got it! the relation to the comment above was missing. Ok so we have unncessery dom cause of media query use instead of the JS solution already in place. ok i would if possible use then the `DeviceStateObserver` rather then adding a new solution and extra DOM. But is not a PR stopper for me

Reply:
It's an ok discussion, DeviceStateObserver vs additional DOM. I argue not to use js if we don't need. Additional DOM is not expensive and the DeviceStateObserver is just overkill.